frameworks/MLPlan/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
+VERSION=${2:-"latest"}
+
+echo "Setup ML-Plan for version $VERSION"
+
+# creating local venv
+. $HERE/../shared/setup.sh $HERE
+
+if [[ -x "$(command -v apt-get)" ]]; then
+    echo "setup system packages"
```

Comment:
for some reason, it failed downloading openjdk without this update, which I don't completely understand because there's already an update before installing python, I think it's a docker thing, each `RUN` command being somehow isolated. Note that we do the same for H2OAutoML and AutoWeka, which also require a jdk.
```suggestion
    echo "setup system packages"
    SUDO apt-get update
```